api/scpca_portal/models/project.py

```diff
@@ -1213,25 +1213,22 @@ def combine_metadata(self, updated_samples_metadata, libraries_metadata, sample_
                 modality=modality,
             )
 
-            sample_libraries_mapping = self.get_sample_libraries_mapping(
-                libraries_metadata[modality]
+            pre_filtered_samples_metadata = (
```

Comment:
Would you be alright with `unfiltered_samples_metadata` or `input_samples_metadata`

Reply:
Sure. I actually originally used `input_samples_metadata`, but was afraid that the name was too generic and that it could be confused with the different usage of `input_samples_metadata` used elsewhere throughout in the file. I like `unfiltered_samples_metadata`.